packages/lib/src/vis/utils.ts

```diff
@@ -35,14 +34,6 @@ export const CAMERA_TOP_RIGHT = new Vector3(1, 1, 0);
 
 const AXIS_OFFSETS = { vertical: 72, horizontal: 40, fallback: 16 };
 
-export const DEFAULT_INTERACTIONS: Interactions = {
-  Pan: true,
-  Zoom: true,
-  XAxisZoom: { modifierKey: 'Alt' },
-  YAxisZoom: { modifierKey: 'Shift' },
-  SelectToZoom: { modifierKey: 'Control' },
-};
```

Comment:
Moved to `interactions/utils` as it is used in `interactions/DefaultInteractions` now